packages/eslint-config/index.js

```diff
@@ -124,5 +124,32 @@ module.exports = {
     'react/display-name': 'off',
     'react-hooks/rules-of-hooks': 'error',
     'react-hooks/exhaustive-deps': 'warn',
+    'import/order': [
+      'error',
+      {
+        'newlines-between': 'always',
+        pathGroupsExcludedImportTypes: ['react'],
+        pathGroups: [
+          {
+            pattern: 'react',
+            group: 'external',
+            position: 'before',
+          },
+          { pattern: 'src/**', group: 'sibling' },
+        ],
+        alphabetize: {
+          order: 'asc',
+          caseInsensitive: true,
+        },
+      },
+    ],
+    'no-restricted-imports': [
+      'warn',
+      {
+        patterns: [
+          'src/*', // Typescript doesn't change import paths, which leads to broken .d.ts files without bundling. Use relative.
+        ],
+      },
+    ],
```

Comment:
Are you're suggesting that people do not use the `src` alias that we provide? Could you expand on the comment you've made here, maybe with an example?

Reply:
Look at `packages/api/dist/auth/index.d.ts` for example. `L29` the variable `context` is any, because `GlobalContext` can't be resolved by typescript without the knowledge of the `tsconfig` path mapping. The problem is that `tsc` does interpret `src/foo` as valid for typechecking, but when generating output, it'll actually not change the path, so there'll be no way for the user's `tsc` to know where that `src/foo` actually points to.

Reply:
🤦 Oh wow, that's pretty lame.

Ok! We share this configuration with consumer's of Redwood. Could you add this rule in `./package.json` instead of here?

Since consumers of Redwood are not going to be distributing typeDefs it should be OK for them to use the `src` alias.

Reply:
Oh shoot, got it, didn't pay much attention to why this eslint config is not just an `.eslintrc` 😅 
I'll move it in a bit.

Reply:
Done

Reply:
Thanks!